plutus-pab/test-node/testnet/pab-config.yml

```diff
@@ -46,6 +46,9 @@ signingProcessConfig:
 metadataServerConfig:
   mdBaseUrl: http://localhost:9085
 
+developmentOptions:
```

Comment:
This is a random bug fix I needed to test `plutus-pab/test-node`. Should be moved to a separate PR if this one isn't wanted.